scipy/stats/_distr_params.py

```diff
@@ -54,6 +54,11 @@
     ['invweibull', (10.58,)],
     ['johnsonsb', (4.3172675099141058, 3.1837781130785063)],
     ['johnsonsu', (2.554395574161155, 2.2482281679651965)],
+    ['kappa4', (-0.1, 0.1)],
+    ['kappa4', (0.0, 0.1)],
+    ['kappa4', (-0.1, 0.0)],
+    ['kappa4', (0.0, 0.0)],
```

Comment:
I think it's appropriate, yes. Something else has multiple entries, is it hypegeometric.